internal/controller/storageclient_controller.go

```diff
@@ -447,8 +447,17 @@ func (r *storageClientReconcile) deletionPhase(externalClusterClient *providerCl
 		}
 	}
 
+	job := &batchv1.CronJob{}
```

Comment:
here var `job` is incorrect, either `cronjob` or `statusReporter` would be good.

Reply:
Changed